core/src/main/scala/com/typesafe/tools/mima/core/MemberInfo.scala

```diff
@@ -7,8 +7,9 @@ object MemberInfo {
  /** The index of the string $_setter_$ in this string */
   private def setterIdx(name: String) = name.indexOf(setterTag)
 
-  private val setterTag = "$_setter_$"
+  private val setterTag = "$_setter_"
```

Comment:
well, this is amusing. because the method in the test is specifically named `bar`, that's somehow being interpreted as a `|` character name-mangled as `$bar`. heh